core_lib/src/canvaspainter.cpp

```diff
@@ -454,12 +454,14 @@ void CanvasPainter::paintCameraBorder(QPainter &painter)
     QRect boundingRect;
     mCameraRect = cameraLayer->getViewRect();
 
+    QRegion rg2(mCameraRect);
```

Comment:
The camera border has always rotated with the *view* but never with the *camera*. Let me demonstrate.
Before 82e6837:
![Peek 2019-09-28 14-27](https://user-images.githubusercontent.com/3461051/65822147-6be9bf80-e1fc-11e9-86dd-fcc0672f0d10.gif)

After 82e6837:
![Peek 2019-09-28 14-29](https://user-images.githubusercontent.com/3461051/65822148-71dfa080-e1fc-11e9-8de7-444ae60f418e.gif)

Based on your picture it looks like it might be applying the view transform twice or something like that, which I did encounter in my testing, but I thought I fixed. I will look into it again.

Reply:
Alright, it should be fixed now. The zoom now should behave exactly as it did before, except with the rotation fixed like above. Note that the zoom will be affected by both the current view and the camera, I just wasn't inverting the transformation. Here's another gif showing how it should behave when zooming out and rotating on the camera layer and in the view (featuring more quality artwork from yours truly :wink: ):

![border_behavior](https://user-images.githubusercontent.com/3461051/67265731-aea15f00-f46b-11e9-945b-b2d856e63870.gif)